### DISCUSSION TITLE
Print documents stuck on generating PDF

### DISCUSSION THREAD

**Discussion by msoroka** [pimcore/pimcore#12615] [Q&A / Support]
Hi, I've created for test purposes print documents from demo. According to [generating PDFs from docs](https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Rendering_PDFs.html) it works fine, but when I crate print documents it stucks on "Genereting PDF" and in networks there are multiple responses like
`{"activeGenerateProcess":true,"date":null,"message":null,"downloadAvailable":false,"statusUpdate":{"status":0,"statusUpdate":"prepare_pdf_generation"}}`
There is nothing wrong in logs. Do you have any ideas how to fix it? 

Pimcore version 10.4.5, PDFreactor. On other instances works fine.

**Comment by dvesh3** [comment]
maybe this helps https://github.com/pimcore/pimcore/issues/10655#issuecomment-947378454
